rsql/rcache.go

```diff
@@ -62,7 +74,8 @@ func (c *rcache) tailUnsafe() int64 {
 func (c *rcache) Load(ctx context.Context, dbc *sql.DB,
 	prev int64, lag time.Duration,
 ) ([]*reflex.Event, error) {
-	if res, ok := c.maybeHit(prev+1, lag); ok {
+	res, ok := c.maybeHit(prev+1, lag)
+	if enabled && ok {
```

Comment:
If the cache isn't enabled we don't even need to call maybeHit. Suggest 

```go
if !disableCache {
  return c.readThrough(ctx, dbc, prev, lag)
}
```

Reply:
I was trying to avoid using negatives as its generally seen as less readable

Reply:
I think the zero value (false) should be the default. If you want to override that thing, it should be true. BTW I got the logic wrong - that should be `if disableCache {}`.

Reply:
Havn't gone gone quite with this recommendation as I wanted the checks to be the same both outside of `readThrough` and the check inside `readThrough`